indico/modules/events/papers/templates/management/overview.html

```diff
@@ -11,22 +12,35 @@
     {% endif %}
 
     <div class="action-box">
-        <div class="section semantic-text {{ 'highlight' if event.cfp.content_reviewing_enabled }}">
+        <div class="section semantic-text {{ 'highlight' if cfp.content_reviewing_enabled }}">
             <div class="icon icon-file-content"></div>
             <div class="text">
                 <div class="label">{% trans %}Content reviewing{% endtrans %}</div>
-                {% if event.cfp.content_reviewing_enabled %}
+                {% set content_deadline = cfp.paper_reviewing_settings.get(event,'content_reviewer_deadline') %}
```

Comment:
Also, why not expose the deadline through a `SettingProperty` on the CFP object?